source/core/src/main/com/csse3200/game/components/tasks/CurrencyTask.java

```diff
@@ -21,7 +21,7 @@ public class CurrencyTask extends DefaultTask implements PriorityTask {
     private long endTime;
     private int interval;
     private final Scrap scrap = new Scrap(); // currency to update
-    private final int currencyAmount = scrap.getAmount(); // amount of currency to update
+    private final int currencyAmount = 30; // amount of currency to update
```

Comment:
as with my other comment, could this be modified to fetch the scrap increment value from the config file?